docs/source/user_guide/model_understanding.ipynb

```diff
@@ -27,7 +27,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First, let's train a pipeline on some data."
+    "First let's train a pipeline on some data."
```

Comment:
lol this is a nit, but I think the comma after the intro word is correct

Reply:
I am pretty sure both are acceptable. However, I don't always grammar real good. Will revert.